lib/spack/spack/spec.py

```diff
@@ -1440,6 +1440,12 @@ def __init__(
         for h in ht.hashes:
             setattr(self, h.attr, None)
 
+        # dictionary of source artifact hashes, set at concretization time
```

Comment:
I think this comment is copied from next attr (and does not accurately describe what `_package_hash` is).

Reply:
fixed this one